backend/lib/calculate/adequacy.py

```diff
@@ -15,14 +15,16 @@
 ONE_MILE_IN_METERS = 1609.344
 ONE_METER_IN_MILES = 1.0 / ONE_MILE_IN_METERS
 # Adequacy results will be inaccurate with smaller RELEVANCY_RADIUS_IN_METERS values.
-RELEVANCY_RADIUS_IN_METERS = 80.0 * ONE_MILE_IN_METERS
+RELEVANCY_RADIUS_IN_METERS = 300.0 * ONE_MILE_IN_METERS
```

Comment:
why are we updating this?

Reply:
This is a good update. When we generate the CSVs we currently have a lot of 10**42s, indicating that no providers were nearby. This relaxes the definition of "nearby" to help the CSVs contain more meaningful info.